modules/videoio/src/cap_v4l.cpp

```diff
@@ -1764,6 +1797,20 @@ static int icvSetPropertyCAM_V4L( CvCaptureCAM_V4L* capture,
         capture->convert_rgb = bool(value) && possible;
         retval = possible || !bool(value);
         break;
+    case CV_CAP_PROP_FOURCC:
+        {
+	        __u32 old_palette = capture->palette;
```

Comment:
Sorry about that, IDE feature for converting indentation tabs to spaces misfired here somehow. Fixed now.